### ISSUE TITLE
Create an ignore list for urlmaps/slugs

### ISSUE THREAD

**Issue by wezell** [dotCMS/core#26263]
### Parent Issue

Right now it is possible to create a urlmap that has a pattern like this `/{urlMap}`.  This has the unfortunate side effect of forcing a regex match and heavy duty ES query for a every request. 

This issue compounds when there are multiple content types that use "root" urlmaps.

We need to create a list of glob patterns that that will be ignored when doing urlMap resolution.  These would include
`/application/**`, `/api/**`, `/dA/**`,  `/dotAdmin/**`, `/html/**`

 

### Problem Statement

Right now it is possible to create a "root" urlmap that has a pattern like this `/{urlMap}`.  This has the unfortunate side effect of forcing a regex match and heavy duty ES query for a every request.  Customers mistakenly add "root" urlMaps/slugs to their sites and in some limited cases, this is ok, where the dotCMS is not loaded or is fronted by a CDN, but it can often lead to server slowdowns. This issue compounds when there are multiple content types that use "root" urlmaps.

One thing we can do to prevent these types of slowdowns is to short circuit the URLMap lookup if the uri matches known system patterns which should never match a url map. 

We need to create a list of glob patterns that that will be ignored when doing urlMap resolution.  These would include
`/application/**`, `/api/**`, `/dA/**`,  `/dotAdmin/**`, `/html/**`

 This includes things like

### Steps to Reproduce

In the dotCMS starter site - run an ab test on a resource in the `/application` folder and save your results.

Hit this url a few times to warm up cache:

https://local.dotcms.site:8443/application/themes/travel/css/styles.dotsass

Then run this

```
ab -n 10000 -c 20 https://local.dotcms.site:8443/application/themes/travel/css/styles.dotsass
```
Now change a few of your url maps to be "root" urlmaps and re-run the `ab` test.  It should perform worse than the test above.

### Acceptance Criteria

URLmaps should not force an ES lookup.

### dotCMS Version

master

### Proposed Objective

Application Performance

### Proposed Priority

Priority 2 - Important

**Comment by swicken-dotcms** [comment]
@wezell can you relate your PR and close this issue?

**Comment by wezell** [comment]
Fixed in: #26265
